### PR TITLE
Nav Unification: introduce Classic Dark as the new default color scheme

### PR DESCRIPTION
**Pull request by frontdevde**
#### Changes proposed in this Pull Request

This PR
* introduces Classic Dark as the new default color scheme
* adds Classic Bright as a copy of the previous default color scheme
* adds missing colors to Classic Blue that were previously inherited from Classic Bright default

Context: 

With the nav unification project we're introducing unified default colors across Calypso and wp-admin. @rickybanister suggested we introduce these colors to Calypso users early via a new default color scheme. The colors in this PR are based on the most recent Navigation Unification, Design Update (see p2-pbAPfg-13V).

Related issue: https://github.com/Automattic/wp-calypso/issues/47135

new-default-color-scheme

#### Testing instructions

* Check out branch locally, `yarn && yarn start`, visit http://calypso.localhost:3000/me/account
* OR use the [calypso.live link](https://calypso.live/?branch=feature/new-default-color-scheme) (easier!)
* Test on a user account that has never had a color scheme set. On this account the new Classic Dark color scheme should show as default now.
* Navigate around Calypso and see if everything looks as expected
* Change color schemes to Classic Bright and Classic Blue and see if those still match what you see in production.

Fixes https://github.com/Automattic/wp-calypso/issues/47135

### CODE REVIEW

**Comment by frontdevde** [comment]
CC'ing all designers involved for design review @sfougnier @lcollette @davemart-in @rickybanister 

Given that this would replace the default color scheme, I'd highly appreciate if y'all could take a look and let me know about any changes we'd want to make. Thank you!

File: packages/calypso-color-schemes/src/shared/color-schemes/_default.scss [hunk 1]
```diff
@@ -252,11 +252,11 @@
...
 
-	--color-masterbar-background: var( --studio-blue-60 );
-	--color-masterbar-border: var( --studio-blue-70 );
+	--color-masterbar-background: var( --studio-gray-100 );
+	--color-masterbar-border: #333;
 	--color-masterbar-text: var( --studio-white );
-	--color-masterbar-item-hover-background: var( --studio-blue-70 );
-	--color-masterbar-item-active-background: var( --studio-blue-90 );
+	--color-masterbar-item-hover-background: #333;
+	--color-masterbar-item-active-background: #23282d;
```
Comment (frontdevde) [inline packages/calypso-color-schemes/src/shared/color-schemes/_default.scss:259]: @sfougnier @lcollette 
These two colors (#333, #23282d) are the exact values from wp-admin. If I understand the new Navigation Unification, Design Update (see p2-pbAPfg-13V) correctly, it seems like for brand consistency we're starting to tweak wp-admin colors a bit too. If so could you let me if these should be replaced with color studio vars and if so which ones? Thank you!

**Comment by davemart-in** [comment]
> Given that this would replace the default color scheme for everyone that doesn't have color scheme set

If possible, I'd very much prefer to make this swap for everyone who signs up past the timestamp for when we launch this. Otherwise this is going to change the color scheme for hundreds of thousands of customers, perhaps millions with no warning or explanation as to why it happened. 

I know introducing a signup date stamp will introduce complexity on the code side of things, but this is the only real option that I see.

**Comment by marekhrabe** [comment]
Next steps as I understand them from the meeting:
- let's add this new color scheme to the list of available ones
- add a separate task for the default scheme choice which will happen under TBD contitions (we can repost the start of the discussion from here)

**Comment by cpapazoglou** [comment]
It seems to me that Classic Dark will be the default for **all users with no specific scheme selected** _after_ nav unification is launched. Till then we can launch it but not make it default. I am not sure we should introduce any signup date timestamp.

**Review by cpapazoglou — CHANGES_REQUESTED** [review]
Thanks @frontdevde for wrangling this!

Followed the testing instructions and LGTM! 
- Booting calypso with a user with no color scheme selected, defaulted to Classic Dark.
- Navigated around and everything seems good. 
- Changing to Classic Bright or Classic Blue works as expected

Following this comment https://github.com/Automattic/wp-calypso/pull/47151#issuecomment-722488401, I would suggest making Classic Bright the default again and creating a separate file for Classic Dark. When we launch nav unification to 100% of users we can move the Classic Dark styles to the default and remove the separate Classic Dark file.

**Comment by frontdevde** [comment]
Closing this PR as the approach changed. Will open a new PR to address https://github.com/Automattic/wp-calypso/issues/47266.

**Pull request closed without merging**